content/en/synthetics/_index.md

```diff
@@ -1,80 +1,70 @@
   tag: "Blog"
-  text: "Introducing Datadog Synthetics"
-- link: "/synthetics/api_tests/"
-  tag: "Documentation"
-  text: "Configure an API Test"
-- link: "/synthetics/browser_tests/"
+  text: "Introduction to Datadog Synthetics"
+- link: "https://www.datadoghq.com/blog/browser-tests/"
+  tag: "Blog"
+  text: "User experience monitoring with Datadog Browser Tests"
+- link: "https://www.datadoghq.com/blog/test-creation-best-practices/"
+  tag: "Blog"
+  text: "Best practices for creating end-to-end tests"
+- link: "/synthetics/guide/"
   tag: "Documentation"
-  text: "Configure a Browser Test"
-- link: "/synthetics/identify_synthetics_bots/"
+  text: "Synthetics Guides"
+- link: "/synthetics/troubleshooting/"
   tag: "Documentation"
-  text: "Identify Synthetics Bots"
+  text: "Troubleshooting Synthetics"
 ---
 
-## Overview
-
-Datadog Synthetics gives you a new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests and browser rendering, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
-
-{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" >}}
+{{< img src="synthetics/synthetics_home.png" alt="Synthetics home page" >}}
 
-## Search
+Synthetic tests allow you to observe how your systems and applications are performing using **simulated requests and actions from around the globe**. Datadog **tracks the performance of your webpages and APIs** from the backend to the frontend, and at various network levels (HTTP, TCP, SSL) in a controlled and stable way, alerting you in case of faulty behavior (regression, broken feature, high response time, unexpected status code, etc.). **End-to-end testing production and CI environments** increases development teams’ speed as it puts an end to the fear of defective code making it to production. **Computing SLOs** on your key endpoints and user journeys makes it easier to stick to your application performance targets and ultimately provide a consistent customer experience. 
 
-Advanced search lets you query checks by any combination of checks attributes:
+## Getting Started
+Create your first Synthetic test and start monitoring web applications to improve their performance in just a few minutes.
+### Set up API tests
 
-* `title` and `message` - text search
-* `status` - Alert, No Data, Ok
-* `creator`
-* `region`
-* `muted`
-* `notification`
-* `tags`
+API tests allow you to launch requests and perform verifications on your web applications at various network levels: HTTP, TCP, SSL. Create your first [HTTP test][1], [TCP test][link to regular documentation], and [SSL test][link to regular documentation] to get started with API and network monitoring.
```

Comment:
No available documentation for TCP tests for now (I need to create it) but we can link to https://docs.datadoghq.com/synthetics/api_tests/?tab=ssltest for SSL tests